NGitLab.Tests/SnippetsTest.cs

```diff
@@ -73,5 +74,57 @@ public async Task Test_snippet_inProject(VisibilityLevel visibility)
 
             snippetClient.Delete(newSnippet.ProjectId, returnedProjectSnippet.Id);
         }
+
+        [TestCase(VisibilityLevel.Private)]
+        [TestCase(VisibilityLevel.Internal)]
+        [TestCase(VisibilityLevel.Public)]
+        [NGitLabRetry]
+        public async Task Test_snippet_files(VisibilityLevel visibility)
+        {
+            using var context = await GitLabTestContext.CreateAsync();
+            var project = context.CreateProject();
+            var snippetClient = context.Client.Snippets;
+
+            var guid = Guid.NewGuid().ToString("N");
+            var snippetName = "testSnip" + guid;
+
+            // arrange
+            var newSnippet1 = new SnippetCreate
+            {
+                Title = snippetName,
+                Visibility = VisibilityLevel.Public,
+                Files = new[]
+                {
+                    new SnippetCreateFile() { Action = SnippetCreateFile.ActionType.Create, FilePath = "Path1.txt", Content = "Content1" },
+                    new SnippetCreateFile {  Action = SnippetCreateFile.ActionType.Create, FilePath = "Path2.txt", Content = "Content2" },
+                },
+            };
+
+            // act - assert
+            snippetClient.Create(newSnippet1);
+            var snippet = snippetClient.All.First();
+
+            Assert.IsNotNull(snippet.Files);
+            Assert.IsNotEmpty(snippet.Files);
+            Assert.IsTrue(snippet.Files.Length == 2);
+
+            Assert.IsNotNull(snippet.Files[0]);
+            Assert.IsTrue(string.Equals(snippet.Files[0].Path, "Path1.txt", StringComparison.Ordinal));
+        }
+
+        [Test]
+        public void  Test_snippet_deserialize()
```

Comment:
I'm not sure this test is very useful as the other one already tests deserialization.